app/src/main/kotlin/com/wire/android/ui/newauthentication/login/NewLoginContainer.kt

```diff
@@ -180,8 +197,8 @@ private fun PreviewNewLoginHeaderNoNavigateBack() = WireTheme {
 private fun PreviewNewLoginHeaderTwoLines() = WireTheme {
     NewLoginHeader(
         title = {
-            NewLoginTitle("Enter your password to log in")
-            NewLoginTitle("Enter your password to log in")
+            NewLoginSubtitle("Enter your password to log in")
+            NewLoginSubtitle("Enter your password to log in")
```

Comment:
shouldn't here be only 1 NewLoginSubtitle? 🤔

Reply:
it's just for the preview to simulate situation when there are multiple texts inside the `NewLoginHeader` to make sure that the navigation icon button is placed properly 😃